source/sec_community_global.ptx

```diff
@@ -4,16 +4,7 @@
     <title>The Challenges of Global Community</title>
 
     <introduction>
-      <p>An open source license formalizes what people are permitted to do with the material that is open source and under what conditions. For example, this might address whether you are granted the rights to use, modify, or distribute (commercially or non-commercially) the material.
-      </p>
-      <p>
-      Disclaimer: It is not the goal of this chapter to provide legal advice. The goal of this chapter is to provide a starting point to help you make an informed choice by providing information on popular open source licenses in a simplified fashion. If you have any questions regarding the best license for your code or any other legal issues relating to it, it’s your responsibility to do further research or consult with a professional.
-      </p>
-      <p>
-      Many people find licensing to be one of the more confusing aspects of open source software. It is our hope that this simplified treatment has given the reader a better understanding of the key issues.
-    </p>
```

Comment:
I deleted these three paragraph since they seem to be out of place, but I think that with minor changes (in the third paragraph) they would fit alright at the beginning of Chapter 5 (Open Source Licensing).

Reply:
@gkaracha Thanks! I have no idea how I managed to get those into the wrong chapter!

Reply:
See 68925028830ac5a1069a29a1c125bcc040202507